cs6232-g4/View/ViewMbrTransactionHistoryForm.cs

```diff
@@ -238,6 +243,23 @@ private void CreateReceipt(double returnTxnId, double refund_or_fine)
             this.ResetFields();
         }
 
+        /// <summary>
+        /// creates the return transaction in the database 
+        /// </summary>
+        private void CreateReturnTransaction()
```

Comment:
you don't need to use this file. It already handles creating a return transaction, what's the use of this method?